pyodide-build/pyodide_build/tests/test_cli.py

```diff
@@ -127,3 +132,52 @@ def test_config_get(cfg_name, env_var):
     )
 
     assert result.stdout.strip() == common.get_make_flag(env_var)
+
+
+def test_fetch_or_build_pypi(selenium, tmp_path):
```

Comment:
There is already a test named the same way on main. I wonder if it's badly merging with `main` in CI and we end up with 2 identically named tests one of which would be ignored. This would explain why coverage says it's not run in CI.

Could you merge the main in, please to see if it would help?

Reply:
Merged main to resolve conflict. Otherwise, nevermind my comment, I did see the function test on main locally, but not after syncing with upstream. Still unclear why codecov doesn't see coverage here, but merging anyway.

Reply:
Yeah, this test was moved to [test_pypi.py](https://github.com/pyodide/pyodide/blob/87ea5862c4a154bcd0277eae40cb566c3f0edbc3/pyodide-build/pyodide_build/tests/test_pypi.py#L212) in #3310. I think I made a mistake while merging main.